docs/UserGuide.md

```diff
@@ -723,11 +727,63 @@ Step by step guide:
     |___________|_____________________________________|_____________________________________________|
     | 1         |  XX                                 |  A                                          |
     |___________|_____________________________________|_____________________________________________|
-        
+*Note: All commands above are not case sensitive.*
+
+#### 4.5.4. Edit Performance List
+Edit the list of students' result under a certain event.  
+This is a step by step command and you may follow the instructions given by the console. 
+During the process, you can choose to edit either student's name or result.
+
+Format: `performance edit`
+    
+Step by step guide to edit name: 
+
+    >>> performance edit
```

Comment:
`Performance edit` guide might need to add in the extra spacing to fit our current tp